src/index.html

```diff
@@ -3,11 +3,261 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>The MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/photos/THEMET.png"
+      type="image/png">
+    <!-- <link rel="stylesheet" href="./styles/main.scss"> -->
+    <link rel="stylesheet" href="/dist/css/main.css">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>The MET</h1>
+    <!-- <div class="container"> -->
+      <div class="met__bar" id="menu">
+        <div class="met__nav">
+          <div class="met__logo">
+            <img src="./images/photos/THEMET.png" alt="logo">
+          </div>
+          <a href="#" class="met__cancel">
+            <img src="./images/cross.svg" alt="cross">
+          </a>
```

Comment:
this element does not display on the menu
![Image 2022-04-13 11 30 07](https://user-images.githubusercontent.com/39308717/163134390-1ba5962d-37a9-4f5b-a6f3-c7ce8d75714e.png)

Reply:
cross.svg